docs/modules/ROOT/pages/vshn-managed/keycloak/customization.adoc

```diff
@@ -0,0 +1,28 @@
+= Customization
```

Comment:
Shouldn't we be more specific about what customizations we're talking about? Is it about custom configuration? Or a custom Keycloak image?

Reply:
```suggestion
= Custom Themes and Providers
```

Reply:
BTW: Why is it needed to have custom themes this way? I thought we have a custom extension by Inventage which makes it possible to upload themes without having to have a custom image? https://products.vshn.ch/appcat/keycloak.html#_configurable_theme If this isn't the case, do we have to update the product description?

Reply:
> Shouldn't we be more specific about what customizations we're talking about? Is it about custom configuration? Or a custom Keycloak image?

I expect that we will have more custom things for Keycloak in the future thus the reason of the main vague title. As a subtitle I aded `Themes and providers`

Reply:
They have a theme editor as well. However, the capabilities are very limited. You can define some colors and an image (that is accessible somewhere on the internet) that you can specify for the welcome theme.
There is currently no option to specify more detailed themes. Also it's not possible to use this to adjust the look and feel when you are logged in (only the welcome page).
We therefore also support providing native keycloak themes.